tests/integration/sync/test_sync_watch.py

```diff
@@ -67,11 +65,15 @@ def setUp(self):
         self.stacks = []
         self.s3_prefix = uuid.uuid4().hex
         self.test_dir = Path(tempfile.mkdtemp())
+        self.template_before = "" if not self.template_before else self.template_before
```

Comment:
What would be the case when `self.template_before` would be an empty string?

Reply:
It should always be defined in the first place. I just initialized it here to appease the IDE.